backend/src/api/routes/oauth-apps/index.routes.ts

```
@@ -0,0 +1,89 @@
+    const apps = await oauthService.listOAuthApps();
+    successResponse(res, apps);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// POST /api/oauth-apps
+router.post('/', verifyAdmin, async (req: AuthRequest, res: Response, next: NextFunction) => {
+  try {
+    const { name, website, redirect_uris } = req.body;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)